File: 백준/Bronze/2750. 수 정렬하기/수 정렬하기.cs
using System;
using System.Collections.Generic;

class Program
{
    static void Main()
    {
        int number = int.Parse(Console.ReadLine());
        List<int> array = new List<int>();
        
        for(int i=1; i<=number; i++)
        {
            array.Add(int.Parse(Console.ReadLine()));
        }
        array.Sort();
        
        string answer = "";
        for(int i=0; i<array.Count; i++)
        {
            answer += array[i] + "\n";
        }
        Console.WriteLine(answer);
    }
}

File: 백준/Bronze/2750. 수 정렬하기/수 정렬하기.js
let input = require('fs')
.readFileSync('/dev/stdin')
.toString().split('\n');

let number = Number(input[0]);
let array = [];
for (let i=1; i<=number; i++)
{
    array[i-1] = input[i].split('\n');
}

    
array.sort(function(a,b){
    return a-b
})

let answer = '';

for (let i=0; i<array.length; i++)
{
    answer += array[i] + "\n";
}
console.log(answer);
